Data/ByteString/Streaming/Char8.hs

```diff
@@ -676,30 +652,36 @@ newline :: Word8
 newline = 10
 {-# INLINE newline #-}
 
+-- | Promote a vanilla `String` into a stream.
 string :: String -> ByteString m ()
 string = chunk . B.pack . Prelude.map B.c2w
 {-# INLINE string #-}
 
-
+-- | Returns the number of times its argument appears in the `ByteString`.
 count_ :: Monad m => Char -> ByteString m r -> m Int
 count_ c = R.count_ (c2w c)
 {-# INLINE count_ #-}
 
+-- | Like `count_`, but suitable for use with `Streaming.mapped`.
```

Comment:
Perhaps usual comment about the `r` continuation. Could that be a note at the bottom referenced from all the relevant functions?